.github/README.md

```diff
@@ -0,0 +1,25 @@
+## How to create a Fedora docker image
+
+Run
+```
+docker login ghcr.io
+```
+
+When prompted use these:
+
+* **User:** the github user
+* **Password:** the github key
```

Comment:
Agree that the instructions are missing words that disambiguate what is currently written. I also see `secrets.DOCKER_CONTAINER_USER_NAME` and `secrets.DOCKER_CONTAINER_REGISTRY_TOKEN`, but it's not clear how to create `DOCKER_CONTAINER_REGISTRY_TOKEN`, and whether that's the key in question or not.

Reply:
I agree that it might not be obvious.
But can't really go and document github "fundamentals"

Anything "secrets.FOO" is to be found in the GitHub secrets area.
In our case https://github.com/unicode-org/icu/settings/secrets/actions

> What is "the github key"?

Sorry, will update.
It is a github token (not key) as documented here: https://cldr.unicode.org/development/maven

It can be typed (with ambition, because it is about 40 char long :-), but can definitely be copy-pasted.

Reply:
I've updated the README.md